terraform/bosh/outputs.tf

```diff
@@ -15,7 +15,19 @@ output "region" {
 }
 
 output "bosh_subnet_id" {
-  value = "${element(split(",", var.infra_subnet_ids), 0)}"
+  value = "${element(split(",", var.infra_subnet_ids), lookup(var.zone_index, var.bosh_az))}"
+}
+
+output "bosh_subnet_cidr" {
+  value = "${lookup(var.infra_cidrs, concat("zone", lookup(var.zone_index, var.bosh_az)))}"
+}
+
+output "default_gw" {
```

Comment:
I think this needs a more specific name to make it clear that it's the default gateway for the Bosh director.